src/Compiler/CompilerUtils.cpp

```diff
@@ -126,6 +124,12 @@ static llvm::cl::opt<std::string> march("march",
     llvm::cl::value_desc("Target a specific architecture type"),
     llvm::cl::cat(OnnxMlirOptions), llvm::cl::ValueRequired);
 
+llvm::cl::list<accel::Accelerator::Kind> maccel("maccel",
+    llvm::cl::desc("Specify an accelerator to generate code for"),
+    llvm::cl::values(clEnumValN(
+        accel::Accelerator::Kind::NNPA, "NNPA", "IBM Telum processor")),
+    llvm::cl::cat(OnnxMlirOptions), llvm::cl::ValueRequired);
```

Comment:
Can we make this optional? For example, it appears only when we build with some accelerators, i.e. `ACCELERATORS_TO_BUILD` is specified. 

If it is impossible, we need to let users know the current driver is not built with accelerator when they specify `--maccel=NNPA`

Reply:
@tungld @etiotto that's what my approach would do